vaultenv.nix

```diff
@@ -0,0 +1,76 @@
+{ haskellPackages
+, mkDerivation
+, lib
+, pkgs
+, glibcLocales
```

Comment:
Are you sure the `glibcLocales` are needed for this derivation?

Reply:
Not sure, but I didn't test whether `vaultenv` still suffers from the `commitAndReleaseBuffer: invalid argument (invalid character)` error message without it, that's why I included this.